scripts/stac/imagery/provider.py

```diff
@@ -14,3 +14,29 @@ class Provider(TypedDict):
     """Organization name"""
     roles: list[ProviderRole]
     """Organization roles"""
+
+
+def merge_provider_roles(providers: list[Provider]) -> list[Provider]:
+    """Merge the roles of the providers with the same name.
+    Does not manage duplicates in the final roles list.
```

Comment:
I don't quite follow what this line means, can you explain to me like I am a 5 year old, please? :laughing:

Reply:
Yeah it's not great explaination. Could you please have another look?